### PR TITLE
user friendliness

### PR DESCRIPTION
**Pull request by tschuy**
Clarify the docs a little and add an file opening error when the user tries to open a bad file.

### CODE REVIEW

File: cinch/cinch [hunk 1]
```diff
@@ -11,8 +11,12 @@ USAGE = """{name} [interpret|dump|vm|compile] file"""
 def main():
     flag = sys.argv[1]
     file_name = sys.argv[2]
-    with open(file_name, 'r') as f:
-        source = f.read()
+    try:
+        with open(file_name, 'r') as f:
+            source = f.read()
+    except IOError as e:
+        print "can't open file {}: {}".format(file_name, e.strerror)
```
Comment (iankronquist) [inline cinch/cinch]: This should print to stderr.

File: cinch/cinch [hunk 2]
```diff
@@ -11,8 +11,12 @@ USAGE = """{name} [interpret|dump|vm|compile] file"""
...
+        sys.exit(1)
```
Comment (iankronquist) [inline cinch/cinch]: This should return the right error from the errno module.

**Comment by iankronquist** [comment]
Looks good with those minor changes.

**Comment by tschuy** [comment]
Updated.

**Comment by iankronquist** [comment]
Looks good to me. Thanks!

**Pull request merged**